java/src/main/java/ai/rapids/cudf/ParquetWriterOptions.java

```diff
@@ -63,9 +72,19 @@ public static Builder builder() {
   private ParquetWriterOptions(Builder builder) {
     super(builder);
     this.statsGranularity = builder.statsGranularity;
+    this.isTimestampTypeInt96 = builder.isTimestampTypeInt96;
   }
 
   public StatisticsFrequency getStatisticsFrequency() {
     return statsGranularity;
   }
+
+  /**
+   * Returns true if the writer is expected to write timestamps in INT96
+   */
+  public boolean isInt96() {
```

Comment:
nit, I would prefer this to be a slightly more descriptive function name. It does not mention timestamp or anything and it is a public method.